### PR TITLE
Feature/783 admin console launcher2

### PR DESCRIPTION
**Pull request by abidingotter**
Clean up the launcher and proxy settings UI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +25,7 @@ dist
 .idea/dictionaries
 .idea/vcs.xml
 .idea/jsLibraryMappings.xml
+.idea/google-java-format.xml
```
Comment (SammyIsConfused) [inline .gitignore]: Is it possible to just gitignore the whole idea folder?

File: src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java [hunk 2]
```diff
@@ -122,6 +124,10 @@ public ClientLauncher() {
 
       ComponentHelper.centreOnScreen(this);
       setVisible(true);
+      JRootPane rootPane = SwingUtilities.getRootPane(launchButton); 
+      rootPane.setDefaultButton(launchButton);
+      //getRootPane().setDefaultButton(launchButton);
```
Comment (SammyIsConfused) [inline src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java]: delete this comment.

File: src/main/java/org/apereo/openequella/adminconsole/service/JarService.java [hunk 3]
```diff
@@ -89,9 +91,10 @@ public void downloadJar(String jarName) {
 				conn.setConnectTimeout(10000);
 				conn.setReadTimeout(10000);
 				conn.setRequestMethod("GET");
+
 				if (metadataFile.exists()) {
 					final JarMetadata meta = JsonService.readFile(metadataFile, JarService.JarMetadata.class);
-					conn.setRequestProperty("If-None-Match", meta.getEtag());
+					//conn.setRequestProperty("If-None-Match", meta.getEtag());
```
Comment (SammyIsConfused) [inline src/main/java/org/apereo/openequella/adminconsole/service/JarService.java]: delete this comment.

File: .gitignore (hunk 1 above)
Comment (abidingotter) [reply .gitignore]: That's what I would prefer, but it was added by @edalex-ian , so I won't touch it.

File: src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java (hunk 2 above)
Comment (abidingotter) [reply src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java]: Oh yeah, I should try to get that to work properly :)

Comment (abidingotter) [reply src/main/java/org/apereo/openequella/adminconsole/launcher/ClientLauncher.java]: Now sets focus to Launch button on window open event if a server is selected

File: .gitignore (hunk 1 above)
Comment (edalex-ian) [reply .gitignore]: I'm not too fussed. My preference is to leave it there, as we can have some project specific bits there and it saves people time. But if it keeps stressing people, then remove it and make sure we ignore it - as it was all the git warnings etc that made me add and then add the ignores.

**Pull request merged**